lanternmq/rabbitmq/messagequeue.go

```diff
@@ -359,9 +357,9 @@ func (mq *MessageQueue) DeclareExchangeReceiveQueue(chID lanternmq.ChannelID, ex
 
 	_, err = ch.QueueDeclare(
 		qName,
-		durableFalse,
+		durableTrue,
 		deleteWhenUnusedFalse,
-		exclusiveTrue,
+		exclusiveFalse,
```

Comment:
I was wondering why these were durableFalse and exclusiveTrue to begin with and if this change would affect anything, but I think I was the one that wrote this initially and arbitrarily, so nevermind, i'm assuming that since the other QueueDeclare functions use durableTrue and exclusiveFalse then it's fine! So I guess this comment is just me rambling lol